index.html

```diff
@@ -6,15 +6,185 @@
+                
+                
+
+            </div>
+
+        </div>
+
+
+
+    </section>
+    <section class="container">
+        <div class="row">
+            <div class="col-12  md-col-6">
+                <div>
+                    <h1>
+                        About Me
+                    </h1>
+                    <p>
+                        Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptatem, ipsam fugiat eaque velit, ullam asperiores minima ipsa quaerat itaque harum eum sint praesentium suscipit? Aliquid iure dolore deserunt laboriosam quis?
+                    </p>
+
+                </div>
+                <div>
+                    <button class="more-btn"> Read More </button>
+
+                </div>
+                <div>
+                    <img class="personal" src="https://www.get-shaped.com/files/personaltrainer-frankufrt.jpg" alt="Personal-photo">
+                </div>
+
+            </div>
+            
+
+        </div>
+
+    </section>
+    <section class="container">
+        <div class="row">
+            <div class="subhero col-12">
+                <div class="heading">
+                    <h1>HOW I CAN HELP YOU</h1>
+
+                </div>
+                <div class="pricing">
+                    <h3>
+                        ONE ON ONE COACHING
+                    </h3>
+                    <h4> Lorem ipsum dolor sit amet consectetur, adipisicing elit. Pariatur exercitationem dolore laborum! Exercitationem eum totam, at maiores, dolor dignissimos eveniet aliquid similique nobis natus iste mollitia odio aspernatur itaque quibusdam.</h4>
+                    <p> 1Hr </p>
+                    <p> £70</p>
+                    <button> Book Now </button>
+
+                </div>
+                <button class="moreserv"> More Services </button>
+
+            </div>
+
+        </div>
+
+    </section>
+
+<footer>
+      <div class="container">
+        <div class="row">
+            <div class="col-12 md-col-6">
+                <P class="contact">
+                    CONTACT
+                </P>
+                <div>
+                    <p>65 Highbury Road, </p>
+                    <p> London SW156SC</p>
+                </div>
+                <div>
+                    <p> Tel:01923 250383</p>
+                    <p> Fax:01923 091278</p>
+                </div>
+                <div>
+                    <p> email@gmail.com</p>
+                </div>
+                
+                <div class="social">
+                        <a href="#" class="fa fa-facebook"></a>
+                        <a href="#" class="fa fa-twitter"></a>
+                        <a href="#" class="fa fa-instagram"></a>
+                        <a href="#" class="fa fa-youtube"></a>
+                </div>
+                
+
 
-    <!-- Add your markup here -->
+            </div>
+     </div> 
 
+    </div>
+    <div>
+        <div class="col-12 md-col-5">
+            <div class="form">
```

Comment:
To make your form more accurate, wrap it with a `form` tag. So all the fields and submit button should be part of a single `form` tag. See example here https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form